src/main/java/eu/openanalytics/containerproxy/auth/impl/SAMLAuthenticationBackend.java

```diff
@@ -67,6 +67,9 @@ public void configureHttpSecurity(HttpSecurity http) throws Exception {
 		.and()
 			.addFilterBefore(metadataGeneratorFilter, ChannelProcessingFilter.class)
 			.addFilterAfter(samlFilter, BasicAuthenticationFilter.class);
+		http
+				.logout()
+				.disable();

```

Comment:
Why is this change required? From my tests, it breaks the logout button.
Even when using Single Logout, the user should be able to logout, right?